src/unix/bsd/mod.rs

```diff
@@ -864,10 +848,6 @@ extern "C" {
         all(target_os = "macos", target_arch = "x86"),
         link_name = "wait4$UNIX2003"
     )]
-    #[cfg_attr(
-        all(target_os = "freebsd", any(freebsd12, freebsd11, freebsd10)),
-        link_name = "wait4@FBSD_1.0"
```

Comment:
Like the others, I think that removing this link_name was unintended.  However, it is coincidentally the right thing to do.  It never should've been there in the first place.  `wait4` hasn't ever been modified since FreeBSD first adopted ELF symbol versions.